conf/config.yaml

```diff
@@ -122,6 +122,7 @@ etcd:
     - "http://127.0.0.1:2379"     # multiple etcd address
   prefix: "/apisix"               # apisix configurations prefix
   timeout: 30                     # 3 seconds
+  version: "v3"                   # etcd version
```

Comment:
I think we can only support etcd v3

Reply:
In my plan, I'll implement for etcd v3 only at first, and then go to multi-version etcd, if necessary. And of course I'll be glad if we support v3 only.